docs/data/date-pickers/custom-field/PickerWithBrowserField.tsx

```diff
@@ -306,8 +368,19 @@ export default function PickerWithBrowserField() {
       <DemoContainer
         components={['DatePicker', 'SingleInputDateRangeField', 'DateRangePicker']}
       >
-        <BrowserDatePicker />
-        <BrowserSingleInputDateRangePicker />
+        <BrowserDatePicker
+          slotProps={{
+            field: { clearable: true },
+          }}
+        />
+        <BrowserSingleInputDateRangePicker
+          slotProps={{
+            field: { clearable: true },
+          }}
+          sx={{
+            width: '100%',
+          }}
```

Comment:
Nitpick: I'd probably try to hide this sx in the `SingleInputField` component instead or at least put it in one line to avoid a somewhat pointless noise. 🤔